R/ci.R

```diff
@@ -1,141 +1,23 @@
 #' Continuous integration setup and badges
 #'
 #' @description
+#' `r lifecycle::badge("questioning")`
```

Comment:
With my recent lifecycle / r-pkgs work, I am reminded of this:

https://lifecycle.r-lib.org/articles/stages.html#superseded-stages

> Superseded stages

> We no longer use these stages, but we document them here because we have used them in the past.

> Questioning ...

Has "questioning" been rehabilitated?

(I'm not deeply invested in this but thought I'd mention.)

Reply:
We have used it a bit in dplyr, so maybe it's more useful than we thought, even if it's not super actionable.